exercise-15/src/com/nayema/Main.java

```diff
@@ -0,0 +1,8 @@
+package com.nayema;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Actor.randomActorName();
```

Comment:
Where is the invocation of the factory method on `Actor`? Why isn't the name of that instantiated object written out here using an `sout`?